features/step_definitions/flowchart_steps.rb

```diff
@@ -0,0 +1,404 @@
+# encoding: utf-8
+require 'pry'
+
+legal_markdown = <<EOF
```

Comment:
I'd probably put this stuff in a text file in /fixtures, and load it in the steps, just to keep this file cleaner